### ISSUE TITLE
More examples

### ISSUE THREAD

**Issue by goodmind** [awto/effectfuljs#5]

**Comment by awto** [comment]
of course, soon, there'll be more examples and blog posts. I'm working on a big update now. After it is finished I'll spend more time on use cases and docs.
